docs/configure_nydus.md

```diff
@@ -1,8 +1,36 @@
 # How to Configure Nydus
 
-Nydus-snapshotter receives a json file as nydusd configuration through CLI option `--config-path`.
+Nydus-snapshotter can receive a toml file as Nudus-snapshotter configuration to startup Nydus-snapshotter
+through CLI option `--config-path`.The `--config-path` can also be set to a json file path as nydusd configuration.
 An example configuration looks like below.
-You can find registry auth from local docker configuration.
+You can refer to the following template for configuration.
+
+```toml
+# Nydus-snapshotter startup parameter
+[snapshotter]
+config_path = "/etc/nydusd-config.json" 
+address = "/run/containerd-nydus/containerd-nydus-grpc.sock"
+root_dir  = "root_dir"
+cache_dir  = "/var/lib/nydus/cache"
+nydusd_binary_path  = "/usr/local/bin/nydusd"
+nydus_image_binary_path = "/usr/local/bin/nydus-image"
+log_to_stdout = true
+...
+# Nydusd default template
+[nydusd.template]
```

Comment:
It seems currently this configuration field is not yet supported, can we remove it?